### PR TITLE
Issues-138 support python 3.9

### PR DESCRIPTION
**Pull request by GordonSo**
Details added to https://github.com/lambci/lambci/issues/138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ different resources yourself.
 
 * Check the [Recipes](#language-recipes) list below on how to configure these:
 * Node.js (any version via [nave](https://github.com/isaacs/nave))
-* Python ([3.8.0, 3.7.4, 3.6.9](#python))
+* Python ([3.9.7, 3.8.12, 3.7.12, 3.6.15](#python))
```
Comment (GordonSo) [inline README.md:62]: These are the latest version of each Python sub-version at the point of creating this PR.

**Review by ChristianMurphy — APPROVED** [review]

**Comment by ChristianMurphy** [comment]
I don't have access to merge either

**Comment by nicoranc** [comment]
Hey @GordonSo, could it be possible for you to work on the merging of this PR? I am working on porting Python3.8 to Python3.9 AWS Lambda functions and cannot use anymore `serverless invoke local` method.

**Comment by GordonSo** [comment]
> Hey @GordonSo, could it be possible for you to work on the merging of this PR? I am working on porting Python3.8 to Python3.9 AWS Lambda functions and cannot use anymore `serverless invoke local` method.

Negative, I don't have access. I have the same problem months ago hence this PR. If you look further into the sibling PR, others have some other suggestions. I'm also not sure what's going on with the state of this repo...

**Comment by andreburto** [comment]
Is there an ETA on this?

**Comment by SceneKim** [comment]
pleaseeeeeeeeee!!!!!!!!!!!!

**Comment by winniewytong** [comment]
@mhart Hi Mike - will you be able to merge this PR? Thanks

**Comment by karkir0003** [comment]
@mhart Pretty please with sprinkles on top, can you merge this PR in? This is blocking me from using Python 3.9 docker image for lambda in `moto`. You will be doing a kind act of service to the developer community

**Comment by mhart** [comment]
I think you might all be confusing this project (which is a CI project built on Lambda) with https://github.com/lambci/docker-lambda

**Pull request closed without merging**